app/models/plant_line.rb

```diff
@@ -19,4 +19,19 @@ class PlantLine < ActiveRecord::Base
                           foreign_key: 'plant_line_name',
                           association_foreign_key: 'plant_population_id'
 
-end
\ No newline at end of file
+  scope :grid_data, ->(ids) do
```

Comment:
Perfect case for a decorator / presenter ;) I won't insist on that but I do not think it should be a scope since it does not return `ActiveRecord::Relation`. It could be a scope with `select` instead of `pluck`.